### ISSUE TITLE
Improve logging button

### ISSUE THREAD

**Issue by grzanka** [yaptide/ui#1416]
Whole row should be clickable, not only icon:

![image](https://github.com/yaptide/ui/assets/7374733/c3473ea0-dad5-4266-bd10-ceaf9b6b81de)

**Comment by matuszsmig** [comment]
@grzanka how should log in/log out button look like when user is logged? 
There is no information about logging out until we hover on this button
![Zrzut ekranu 2024-03-05 233338](https://github.com/yaptide/ui/assets/115647012/05c662f7-f75d-49c0-b74f-8fadd51bf447)

**Comment by grzanka** [comment]
> @grzanka how should log in/log out button look like when user is logged? There is no information about logging out until we hover on this button ![Zrzut ekranu 2024-03-05 233338](https://private-user-images.githubusercontent.com/115647012/310299716-05c662f7-f75d-49c0-b74f-8fadd51bf447.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MjQ3MzksIm5iZiI6MTcwOTcyNDQzOSwicGF0aCI6Ii8xMTU2NDcwMTIvMzEwMjk5NzE2LTA1YzY2MmY3LWY3NWQtNDljMC1iNzRmLThmYWRkNTFiZjQ0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwNlQxMTI3MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MDYwZWE2NTZhMmJhMTdjNTExZjc1OGZhNzdlZDdhODlmYWEzNjVhZDhlZjY3MzlmMjZhZTE5MzU5ZGQwNjI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.sg9kAbiU83y6CATtaCkvlTxEEYoviMWCp64ADtDrjZg)

I would remove the username: replace "admin" with "Log out **admin**" text and use different icon.

**Comment by matuszsmig** [comment]
Right now looks like this: 

![Image](https://github.com/yaptide/ui/assets/115647012/be9664d9-5a92-4b4d-9790-ecae091cdf42)

Which icon should I use?
@grzanka

**Comment by matuszsmig** [comment]
@grzanka I have devided log out message into two lines:
- "Log out"
- user nickname
![image](https://github.com/yaptide/ui/assets/115647012/22d41d06-1d7f-4785-add5-95615c26476a)

In case of too long nickname we have two solutions:
-Behaves normal way if text doesn't fit it not appear
![image](https://github.com/yaptide/ui/assets/115647012/fc8470bd-d9c7-40d2-adb9-f57d82566953)
-We may use text-overflow: ellipsis; and it will wrap into "sometex..."
![image](https://github.com/yaptide/ui/assets/115647012/fc3da73d-1fb0-4c84-9be8-e4ffa83227ee)

Which one solution is better?

**Comment by grzanka** [comment]
Second one is better

**Comment by matuszsmig** [comment]
@grzanka https://github.com/yaptide/ui/pull/1518 is ready for Code Review and then for Merge
